src/main/java/uk/gov/mca/beacons/api/controllers/OwnerController.java

```diff
@@ -34,7 +35,7 @@ public OwnerController(
   @PostMapping
   @ResponseStatus(HttpStatus.CREATED)
   public WrapperDTO<BeaconPersonDTO> createOwner(
-    @RequestBody WrapperDTO<BeaconPersonDTO> dto
+    @RequestBody @Valid WrapperDTO<BeaconPersonDTO> dto
```

Comment:
[See article here](https://www.baeldung.com/spring-boot-bean-validation#implementing-a-rest-controller)

This will enable Spring's default bean validator (Hibernate) and if it fails validation Spring will throw a `MethodArgumentNotValidException` which is what we handle in the `RestResponseEntityExceptionHandler` class.  This is the global exception handler for all controllers